libs/ledger/src/chain/main_chain.cpp

```diff
@@ -1367,6 +1367,9 @@ bool MainChain::DetermineHeaviestTip()
           auto const &a_hash{a.first}, &b_hash{b.first};
           auto        a_weight{a.second.weight}, b_weight{b.second.weight};
 
+          // The weight, which is related to the rank of the miner producing the block is used here
```

Comment:
A little confusing here for newcomers I would suspect. Suggest: 

```
The weight, which is related to the rank of the miner producing the block is used here
to tie break chains with equivalent total weight, choosing the weight of the tips as a tiebreaker.
This is important for consensus.
```

Reply:
Sure.